src/five/customerize/tests.py

```diff
@@ -39,12 +37,6 @@ def tearDown(self):
 
 class Py23DocChecker(doctest.OutputChecker):
```

Comment:
I think the whole Py23DocChecker class can be removed.

Reply:
Good catch!